spring-cloud-starter-stream-processor-groovy-transform/src/test/resources/script-with-grab.groovy

```diff
@@ -0,0 +1,3 @@
+@Grab('org.grooscript:grooscript:1.3.0')
+import org.grooscript.GrooScript
+GrooScript.convert(new String(payload))
```

Comment:
Please, ensure new line in the end of each file.
You can configure your IDE to do that for your on save.